src/check_ast.rs

```diff
@@ -341,6 +341,9 @@ where
                 if self.settings.enabled.contains(&CheckCode::B019) {
                     flake8_bugbear::plugins::cached_instance_method(self, decorator_list);
                 }
+                if self.settings.enabled.contains(&CheckCode::B021) {
+                    flake8_bugbear::plugins::f_string_docstring(self, body);
+                }
```

Comment:
Could we move this (and the class version) to just before `let definition = docstrings::extraction::extract(...)`? That way it'd at least be colocated with the other docstring-tracking code.